src/Model/UserRole.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace WPGraphQL\Model;
+
+
+use GraphQLRelay\Relay;
+
+/**
+ * Class UserRole - Models data for user roles
+ *
+ * @property string $id
+ * @property string name
+ * @property array  $capabilities
+ *
+ * @package WPGraphQL\Model
+ */
+class UserRole extends Model {
+
+	/**
+	 * Stores the incoming user role to be modeled
+	 *
+	 * @var array $user_role
+	 * @access protected
+	 */
+	protected $user_role;
+
+	/**
+	 * UserRole constructor.
+	 *
+	 * @param array $user_role The incoming user role to be modeled
+	 *
+	 * @access public
+	 * @return void
+	 * @throws \Exception
+	 */
+	public function __construct( $user_role ) {
+		$this->user_role = $user_role;
+		parent::__construct( 'UserRoleObject', $user_role, 'list_users', [ 'id', 'name' ] );
```

Comment:
so, can users see their own Role if they don't have `list_users` cap? Seems like the role of the current user should be visible still, eh?